### PR TITLE
[plugin-mobile-app] Introduce a step to navigate back

### PR DESCRIPTION
**Pull request by uarlouski**

### CODE REVIEW

File: docs/modules/plugins/pages/plugin-mobile-app.adoc [hunk 1]
```diff
@@ -399,4 +399,28 @@ When I upload file `images/avatar.png` to device
...
+==== *_Wording_*
+
+[source,gherkin]
+----
+When I navigate back
+----
+
+==== *_Usage_*
+
+.NavigateBack.story
```
Comment (valfirst) [inline docs/modules/plugins/pages/plugin-mobile-app.adoc:417]: `.NavigateBack.story` -> `.Navigate back in the mobile application`

File: docs/modules/plugins/pages/plugin-mobile-app.adoc [hunk 2]
```diff
@@ -399,4 +399,28 @@ When I upload file `images/avatar.png` to device
...
+[source,gherkin]
+----
+Scenario: Go back from login page
```
Comment (valfirst) [inline docs/modules/plugins/pages/plugin-mobile-app.adoc:420]: scenario title should be updated

File: vividus-plugin-mobile-app/src/main/java/org/vividus/bdd/mobileapp/steps/DeviceSteps.java [hunk 3]
```diff
@@ -51,4 +54,13 @@ public void uploadFileToDevice(String filePath)
...
     }
+
+    /**
+     * Moves back a single "item" in the mobile application
+     */
+    @When("I navigate back")
+    public void navigateBack()
+    {
+        webDriverProvider.get().navigate().back();
+    }
```
Comment (valfirst) [inline vividus-plugin-mobile-app/src/main/java/org/vividus/bdd/mobileapp/steps/DeviceSteps.java:65]: https://github.com/vividus-framework/vividus/blob/b4e861232068227f0f3d1cdcf01bf10893225a66/vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/PageSteps.java#L320-L324

**Review by valfirst — CHANGES_REQUESTED** [review]

**Comment by uarlouski** [comment]
Temporarily close

**Pull request closed without merging**